modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/AssetManagerImpl.java

```diff
@@ -1586,7 +1589,7 @@ private Function<Optional<Event>, Optional<Event>> getEventUpdateFunction(Snapsh
       // extended metadata
       event.resetExtendedMetadata();  // getting rid of old data
       for (EventCatalogUIAdapter extendedCatalogUIAdapter : extendedEventCatalogUIAdapters.getOrDefault(orgId,
-              Collections.emptyList())) {
```

Comment:
If an event catalog is for a specific organization and one for all, we would still miss the wildcard one.